liveHomeFrontend/src/components/Input.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+//styles
+import styles from '../styles/components/Input.module.sass'
+
+const Input = (props) => {
+  return (
+    <input 
```

Comment:
the input element doesn´t have a label, in the design we use a label. The focus event is the default i think we should customize it

Reply:
I don't make a label because we define this to be a different component, but I also think this component is more needed in the input component.
So I make the change by adding tag to input component. Thank you